test_header_footer_links.js

```diff
@@ -0,0 +1,62 @@
+//Tests header and footer links for expected behaviour.
+var url = "https://neontabs.neontribe.org/latest/hr/"
```

Comment:
try var url = casper.cli.get("home");

then run from the command line using --home="https://neontabs.neontribe.org/latest/hr"